src/index.html

```diff
@@ -1,12 +1,345 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
```

Comment:
And you could delete container div in line 12, and use instead of it - mix. More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
And add changes to styles according to changes in html.

```suggestion
    <header class="header page__header">
```